satellite-clone-vars.sample.yml

```diff
@@ -62,3 +62,10 @@ org: changeme
 #   - it is the responsibility of the user to make the required repos available.  Otherwise, the script will stop with an error.
 # uncomment the following line to override this option (defaults to true)
 #register_to_portal: true
+
+# This setting determines if the run is a satellite migration or satellite clone
+# If satellite_migration is set to false:
+#   - satellite clone is run
+# If satellite_migration is set to true:
+#   - satellite migration is run
```

Comment:
This makes migration and clone seem like two separate tasks. There is also no mention of what a migration means. Can you specify that this should be set to true if you are using a back up from a RHEL 6 machine and want to clone to a RHEL 7 machine or something similar to that.